hack/helm-reference-gen/main.go

```diff
@@ -54,13 +54,17 @@ var (
 	// And will not match the "# yaml comment" incorrectly.
 	commentPrefix = regexp.MustCompile(`(?m)^[^\S\n]*#[^\S\n]?`)
 
+	funcMap = template.FuncMap{
+		"ToLower": strings.ToLower,
+	}
+
 	// docNodeTmpl is the go template used to print a DocNode node.
 	// We use $ instead of ` in the template so we can use the golang raw string
 	// format. We then do the replace from $ => `.
 	docNodeTmpl = template.Must(
-		template.New("").Parse(
+		template.New("").Funcs(funcMap).Parse(
 			strings.Replace(
-				`{{- if eq .Column 1 }}### {{ .Key }}
+				`{{- if eq .Column 1 }}### {{ .Key }} ((#h-{{ .Key | ToLower }}))
```

Comment:
can you use the `#v{{ .HTMLAnchor }}` ?

Reply:
Do you need this here or only on the top-level heading nodes?

Reply:
We only needed it on the TOC because there is a new subtask for `connectInject` (`global.secretsBackend.vault.connectInject`) which was interfering with the top level linking when the website renders links.